modules/assimp/assimp/config.h

```diff
@@ -0,0 +1,980 @@
+/*
+---------------------------------------------------------------------------
+Open Asset Import Library (assimp)
```

Comment:
This file duplicate `thirdparty/assimp/assimp/config.h` now, no?

Reply:
Removed.